recipe-generator/src/main/java/ca/derekcormier/recipe/generator/JavaScriptCookbookGenerator.java

```diff
@@ -0,0 +1,14 @@
+package ca.derekcormier.recipe.generator;
+
+import ca.derekcormier.recipe.cookbook.Cookbook;
+import ca.derekcormier.recipe.generator.filter.JsParamFilter;
+import ca.derekcormier.recipe.generator.filter.JsValueFilter;
+import liqp.filters.Filter;
+
+public abstract class JavaScriptCookbookGenerator extends CookbookGenerator {
+    public JavaScriptCookbookGenerator(Cookbook cookbook) {
+        super(cookbook);
+        Filter.registerFilter(new JsParamFilter(cookbook));
```

Comment:
This API is odd 🤔 

Invoking a static method with an object created non-staticly?

Reply:
@bkiers :)

Reply:
I think I made it that way at first because it was first private and only was used to add all built-in filters (which could be static since they were always there for every template).

The `Template` class has an instance method [`#with(Filter filter): Template`](https://github.com/bkiers/Liqp/blob/master/src/main/java/liqp/Template.java#L195) that you could possibly use?

HTH

Reply:
Ah, I hadn't noticed the `with` method. Thanks!

Reply:
No problem, also have a look at the README: https://github.com/bkiers/Liqp#22-custom-tags